tools/Containerfile.dev

```diff
@@ -0,0 +1,26 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN dnf install -y @rpm-development-tools git
+
+RUN mkdir /prepare
+WORKDIR /prepare
+
+COPY osbuild.spec .
+COPY tools/install_spec_file_dependencies.sh .
+
+RUN chmod a+x install_spec_file_dependencies.sh
+RUN ./install_spec_file_dependencies.sh
```

Comment:
(quick dirve-by) AIUI, dnf can do this itself (part of dnf-plugins-core iirc) without a script, i.e. something like `dnf builddep ./osbuild.spec` should work. Or is there a downside to that? If so, might be worth a short comment.

Reply:
done, please check